CONTRIBUTING.md

```diff
@@ -502,27 +498,24 @@ To get a sense for what changes are considered mass rebuilds, see [previously me
 ## Commit conventions
 [commit-conventions]: #commit-conventions
 
-- Create a commit for each logical unit.
-
-- Check for unnecessary whitespace with `git diff --check` before committing.
+- Create one commit for each logical unit.
 
 - If you have commits `pkg-name: oh, forgot to insert whitespace`: squash commits in this case.
   Use `git rebase -i`.
   See [Squashing Commits](https://git-scm.com/book/en/v2/Git-Tools-Rewriting-History#_squashing) for additional information.
 
 - For consistency, there should not be a period at the end of the commit message's summary line (the first line of the commit message).
 
-- When adding yourself as maintainer in the same pull request, make a separate
-  commit with the message `maintainers: add <handle>`.
+- When adding yourself as maintainer, make a separate commit with the message `maintainers: add <handle>`.
   Add the commit before those making changes to the package or module.
   See [Nixpkgs Maintainers](./maintainers/README.md) for details.
 
 - Make sure you read about any commit conventions specific to the area you're touching.
   See:
-  - [Commit conventions](./pkgs/README.md#commit-conventions) for changes to `pkgs`.
+  - [Commit conventions](./doc/README.md#commit-conventions) for changes to `doc`, the Nixpkgs manual.
   - [Commit conventions](./lib/README.md#commit-conventions) for changes to `lib`.
   - [Commit conventions](./nixos/README.md#commit-conventions) for changes to `nixos`.
-  - [Commit conventions](./doc/README.md#commit-conventions) for changes to `doc`, the Nixpkgs manual.
+  - [Commit conventions](./pkgs/README.md#commit-conventions) for changes to `pkgs`.
```

Comment:
I get that this is alphabetical, but `pkgs/` is the hot path and should IMO be on top.